.github/workflows/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: 'bug'
+assignees: ''
+
+---
+
+**Bug description**
+A clear and concise description of what the bug is. Please do not forget to add:
+- Orion version: X.Y.Z
```

Comment:
```suggestion
- Orion version: X.Y.Z (this can be get with `GET /version` in the API port or using `contextBroker --version`)
```

Reply:
Added in https://github.com/telefonicaid/fiware-orion/pull/4010/commits/6b153302f407775679adbb469c3cfb064acd76e3